main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nickname, player_info in players_data.items():
```

Comment:
It's a good practice to validate the structure of 'players_data' to ensure it contains the expected keys and data types before processing.